openlibrary/catalog/add_book/__init__.py

```diff
@@ -516,6 +517,32 @@ def update_ia_metadata_for_ol_edition(edition_id):
                     data = item.metadata
     return data
 
+def create_edition_from_amazon_metadata(isbn):
```

Comment:
@hornc you may wish to contend where these helpers live (e.g. should they live somewhere outside of catalog/addbook/__init__.py?